zerver/lib/request.py

```diff
@@ -48,9 +48,10 @@ class _NotSpecified:
         pass
     NotSpecified = _NotSpecified()
 
-    def __init__(self, whence=None, converter=None, default=NotSpecified,
-                 validator=None, argument_type=None):
-        # type: (str, Callable[Any, Any], Any, Callable[Any, Any], str) -> None
+    def __init__(self, whence: str = None, converter: Callable[[Any], Any] = None,
```

Comment:
Should have been `whence: str=None`, and there shouldn't be spaces around the `=` operator for the rest of the cases below.